src/components/WorkspaceFilter/WorkspaceFilter.js

```diff
@@ -164,48 +98,44 @@ const WorkspaceFilter = props => {
-                        <Button className='small-button' id={filter.goodForGroups ? 'clicked' : 'unclicked'} onClick={handleSelect} name='goodForGroups'>Good for Groups</Button>
-                        <Button className='small-button' id={filter.outdoorSpace ? 'clicked' : 'unclicked'}  onClick={handleSelect} name='outdoorSpace'>Outdoor Space</Button>
+                        <Button className='select-button' name='fastWifi' id={setProp(filter.fastWifi)} value={setProp(filter.fastWifi)} onClick={handleSelect}>Fast WiFi</Button>
+                        <Button className='select-button' name='quiet' id={setProp(filter.quiet)} value={setProp(filter.quiet)} onClick={handleSelect}>Quiet</Button >
+                        <Button className='select-button' name='cafe' id={setProp(filter.cafe)} value={setProp(filter.cafe)} onClick={handleSelect}>Cafe</Button >
+                        <Button className='select-button' name='outlets' id={setProp(filter.outlets)} value={setProp(filter.outlets)} onClick={handleSelect}>Outlets Available</Button>
+                        <Button className='select-button' name='food' id={setProp(filter.food)} value={setProp(filter.food)} onClick={handleSelect}>Food</Button>
+                        <Button className='select-button' name='cowork' id={setProp(filter.cowork)} value={setProp(filter.cowork)} onClick={handleSelect}>Co-Work Space</Button>
+                        <Button className='select-button' name='coffee' id={setProp(filter.coffee)} value={setProp(filter.coffee)} onClick={handleSelect}>Coffee</Button>
+                        <Button className='select-button' name='alcohol' id={setProp(filter.alcohol)} value={setProp(filter.alcohol)} onClick={handleSelect}>Beer + Wine</Button >
+                        <Button className='select-button' name='openLate' id={setProp(filter.openLate)} value={setProp(filter.openLate)} onClick={handleSelect}>Open Late</Button >
+                        <Button className='select-button' name='lotsOfSeats' id={setProp(filter.lotsOfSeats)} value={setProp(filter.lotsOfSeats)} onClick={handleSelect}>Lots of seats</Button >
+                        <Button className='select-button' name='library' id={setProp(filter.library)} value={setProp(filter.library)} onClick={handleSelect}>Library</Button >
+                        <Button className='select-button' name='pets' id={setProp(filter.pets)} value={setProp(filter.pets)} onClick={handleSelect} name='pets'>Pet-Friendly</Button>
+                        <Button className='select-button' name='parking' id={setProp(filter.parking)} value={setProp(filter.parking)}  onClick={handleSelect}>Parking</Button>
+                        <Button className='select-button' name='goodForGroups' id={setProp(filter.goodForGroups)} value={setProp(filter.goodForGroups)} onClick={handleSelect}>Good for Groups</Button>

```

Comment:
We can use a ternary statement to set id to `'on'` or `'off'`:
`id={filter.goodForGroups ? 'on' : 'off'}`